secator/runners/command.py

```
@@ -558,6 +559,11 @@ def yielder(self):
 
 		except BaseException as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `BaseException`

(BLE001)